src/Tests/Entities/ProjectTest.php

```diff
@@ -78,7 +79,41 @@ public function testSuccessToAssociativeArray()
             'deadline' => '21/12/2012',
             'overdue' => true
         ];
-        $actualOutput = $testProject->toAssociativeArray();
+        $actualOutput = $testProject->toAssociativeArrayFewerProperties();
```

Comment:
your test method is still called `testSuccessToAssociativeArray`

Reply:
done